### DISCUSSION TITLE
Is it possible for the developers to have it to work with iOS 12?

### DISCUSSION THREAD

**Discussion by Naro323** [mKeRix/room-assistant#747] [Q&A]
I wanted to test this out but my phone doesn't update past iOS12.5.3, and the app says it needs a minimum of iOS 14.0

**Comment by mKeRix** [comment]
The app itself is built with SwiftUI (requires at least iOS 13) and the new logging stack (requires iOS 14). Since I'm not really a Swift/iOS developer and am quite happy with the current SwiftUI setup I won't be investing any time into back porting this app to older iOS versions, sorry.

That being said, the app is open source on GitHub. If you or somebody else is interested in backporting feel free to create a PR!

**Reply by BelgarionNL** [reply]
Bummer I was going to add my ipad air but also does not update to 14...
